multicluster/config/overlays/member/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+namespace: kube-system
```

Comment:
should we also allow user to update this namespace? this is configured via `ClusterSet`, right?

Reply:
for member cluster, it doesnt matter because it has cluster-wide access to resources.